### PR TITLE
Add view

### PR DESCRIPTION
**Pull request by yonas44**
## These are the changes implemented to this PR:
- Add views for the corresponding controller methods.
- Use the methods created from previous project to display `three recent posts` and `five recent comments`.
- Add functionality for the view of `Users` and `Posts`.
- Add some styling to display for the user.

### CODE REVIEW

File: app/views/posts/show.html.erb [hunk 1]
```diff
@@ -1,9 +1,16 @@
...
-    <h4><%= @post.title %></h4>
-    <p><%= @post.text %></p>
-    <p><%= @post.id %></p>
+    <div id="post-container">
+        <div class="post-top-section">
+            <h4>Post #1 by <%= @post.author.name %></h4>
+            <p id="comments-likes">Comments: <%= @post.comments_counter %>, Likes: <%= @post.likes_counter %></p>
+        </div>
+        <p><%= @post.text %></p>
+    </div>
```
Comment (zieeco) [inline app/views/posts/show.html.erb:8]: - Hey, Please, let's kindly make the show page to be close to the wireframe by adding the omitted border to match the wireframe. see these images below:

*wereframe*
Screenshot from 2023-03-02 16-18-02

*your design*
Screenshot from 2023-03-02 16-17-30

**Review by zieeco — CHANGES_REQUESTED** [review]
Hi @yonas44 ,


Great job so far! 👏 
You've done well, however, there are some issues that you still need to work on to go to the next project, but you are almost there!


## To highlight 🎯

- No linter errors.✔️
- Nice design. ✔️
- Descriptive PR. ✔️


## Required Changes ♻️
_Check the comments under the review._





## Optional suggestions
- N/A

**_Every comment with the **[OPTIONAL]** prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._**


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app/views/posts/show.html.erb (hunk 1 above)
Comment (yonas44) [reply app/views/posts/show.html.erb:8]: Okay, thanks for the feedback, I will add it.

**Review by Wes-Isaac — APPROVED** [review]
Hi @yonas44,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- Professional README 👍.
- Requirements met 💯.
- Great communication skill 🥇.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**